assets/helpers/url.js

```diff
@@ -34,10 +56,25 @@ const addQueryParamToURL = (urlOrPath: string, paramsKey: string, paramsValue: ?
   return `${strInit}?${paramsObj.toString()}`;
 };
 
+function getDomain(): Domain {
```

Comment:
Maybe `baseDomain` instead of `getDomain`?

Reply:
Good call, updated.